script.js

```diff
@@ -1,4 +1,218 @@
+
+// function to generate random number
+function random(max) {
+  let randomNum = Math.floor(Math.random() * max) + 1;
+  return randomNum;
+}
+// create function to generate random item
```

Comment:
Consider using JSDoc syntax to comment your functions for best practice. https://jsdoc.app/about-getting-started.html